### DISCUSSION TITLE
Need explanation about Config

### DISCUSSION THREAD

**Discussion by SkyLostTR** [petrleocompel/hikaxpro_hacs#147] [Q&A] [answered]
Greetings,
I got host part but what we put on username/pass field? I tried some and wont work.

**Comment by petrleocompel** [comment, marked as answer]
Well depends on how you set up your system. 

- Cloud setup ?  Your app logins. 
- Local setup ? Your local account

You can check credentials on web interface of your device.

**Comment by SkyLostTR** [comment]
Thank you very much. My own solution was to create more of a partner account. Up to that point, the system wouldn't let me log in even though I tried username, phone, email.
